Server/pom.xml

```diff
@@ -42,6 +42,15 @@
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-dependency-plugin</artifactId>
 			</plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <systemProperties>
+                        <org.apache.commons.logging.Log>org.openas2.logging.Log</org.apache.commons.logging.Log>
+                    </systemProperties>
+                </configuration>
+            </plugin>
```

Comment:
The XML has inconsistent identatoin

Reply:
Sorry, I didn't get you. What is exactly inconsistent?

Reply:
Looks fine to me.